openid-4-verifiable-presentations-1_0.md

```diff
@@ -3188,8 +3228,12 @@ The technology described in this specification was made available from contribut
 
    [[ To be removed from the final specification ]]
 
-   -25
+   -26
+  
+   * add `attachments` to list of authorization parameters
 
+   -25
+   
```

Comment:
```suggestion
   -25
   
   * add `attachments` to list of authorization parameters
```
why is this suddenly -26? what did i miss?

Reply:
I thought because 25 got already published I needed to put it into 26. But I can change it back to 25

Reply:
I reverted that change, it's back in 25

Reply:
I think you were right the first time @cre8 , this should be in -26. Although technically you only became correct this morning when -25 was actually published :)

Reply:
added to v26 again